apps/tangle-dapp/.eslintrc.json

```diff
@@ -6,7 +6,7 @@
     "../../.eslintrc.json",
     "eslint:recommended"
   ],
-  "plugins": ["unused-imports", "simple-import-sort"],
```

Comment:
Is there an issue with this ESLint plugin that makes you want to remove it? I think it helps us organize our imports.

Reply:
While making changes it keeps erroring and I have to keep constantly fixing it, and on this specific PR it wasn't reporting the error for some reason 👀 It does help organize the imports by sorting them, but I would say it's a bit buggy 😅

Reply:
You can use this command on VSCode to fix the issue automatically.

![CleanShot 2024-05-30 at 13 25 42](https://github.com/webb-tools/webb-dapp/assets/60747384/2d24198c-8a12-4351-8bda-df155b4ce2d0)

Reply:
Yes, that's what I currently use. However, the problem is that I have to be doing that constantly as I make changes everywhere 👀